labman/db/process.py

```diff
@@ -2807,8 +2814,10 @@ def _format_sample_sheet(self, data, sep=','):
             'Workflow{sep}{Workflow}\nApplication{sep}{Application}\n'
             'Assay{sep}{Assay}\nDescription{sep}{Description}\n'
             'Chemistry{sep}{Chemistry}\n\n[Reads]\n{read1}\n{read2}\n\n'
-            '[Settings]\nReverseComplement{sep}{ReverseComplement}\n\n'
-            '[Data]\n{data}')
+            '[Settings]\nReverseComplement{sep}{ReverseComplement}\n%s'
+            '\n[Data]\n{data}' % (
```

Comment:
Mixing dict formatting with string formatting is a bit confusing to look at, would it be possible to do this such that the optional string (based off of `is_amplicon`) is part of the formatting dict used when filling the template?

Reply:
Good point! Let me edit.